rsyslog/init.sls

```diff
@@ -50,6 +51,33 @@ rsyslog_mmjsonparse:
     - require:
       - pkg: rsyslog
 
+{%- else %}
+rsyslog_updates:
+  pkgrepo.managed:
+    - name: deb http://ppa.launchpad.net/adiscon/v8-devel/ubuntu focal main
+    - dist: focal
+
+rsyslog:
+  pkg.installed:
+    - name: rsyslog
+    - version: 8.2008.0~20200811150537-0adiscon1focal
```

Comment:
this version number seems a bit different than the ones used for ubuntu 16, is the `~20200811150537` part a commit hash?

Reply:
No idea where that comes from. I went with the latest available by doing `apt list | grep rsyslog` after adding  the `adiscon` repo. Now that I think about it, this could be a dev build so I'll double check.

Reply:
Ahh yeah I was using the `v8-devel` ppa. Switched over to `v8-stable`. The latest version is now `8.2106.0-0adiscon1focal1` which makes more sense.